src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/order_actions.html.twig

```diff
@@ -47,4 +47,6 @@
       {{ form_rest(updateOrderStatusActionBarForm) }}
     </div>
   {{ form_end(updateOrderStatusActionBarForm) }}
+
+  {{ renderhook('displayBackOfficeOrderActions', {'id_order': orderForViewing.id}) }}
```

Comment:
Regarding this hook, I don't know if we can implement "More actions", see below.

![actions](https://user-images.githubusercontent.com/15143151/67657744-57662780-f960-11e9-8bec-90a6eb981208.png)

PrestaShop does not define clear API for this, so modules can return anything (be it a button, multiple buttons or some other elements). What we know is that we get string in the end, but we don't know what's inside.

@matks your thoughts?

Reply:
Expected behavior is "if more than 2 modules are hooked on this button, then we show the 1st 2 buttons and the following buttons are put inside a drop-down "more actions" " is that right ?

Maybe we should drop the drop-down and go for a basic toggle

Reply:
> "if more than 2 modules are hooked on this button, then we show.......

Yup.

> Maybe we should drop the drop-down and go for a basic toggle

I'm afraid it might add unnecessary limitations, because as mentioned before, it can be more than a buttons, maybe form with input or anything else. :thinking:

Reply:
By toggle I meant this: "when I click on "more actions", it shows all previously hidden blocks added by hook modules. So they can be displayed in any way they want.

I think the toggle is more compatible that a dropdown, isn't it ? 🤔

Reply:
Ohhh, but I don't know tbh, let's wait for @TristanLDD feedback. :slightly_smiling_face:

Reply:
I think the question here is about how we want the modules to integrate this page: 
the name of the hook is "display back office order actions" so in my opinion, it should be a hook for action buttons only. If modules want to bring a form with inputs or other components there are more relevant hooks to do that, like display admin order.

Reply:
@TristanLDD thanks for reply, but PrestaShop itself show example of using form here, see https://prnt.sc/pq17mi so I think modules may have similar use cases.

Either way, we get HTML content from modules and it's nearly impossible for us to enforce some structure to it, so we cannot know whether modules returns 1 button or 10 buttons or maybe it's not button at all. :confused: 

So, wdyt? What should we do here? @matks your thoughts?